client/src/pages/dashboard.tsx

```diff
@@ -11,13 +11,17 @@ import { MySQLIcon } from '../ui/icons/MySQLIcon';
 
 export const Dashboard = () => {
   // const history = useHistory();
-  const { data /* loading, error */ } = useDashboardQuery({});
+  const { data, refetch /* loading, error */ } = useDashboardQuery({});
 
   // TODO show loading
   // TODO handle error
 
   // TODO if no apps or dbs show onboarding screen
 
+  useEffect(() => {
```

Comment:
not a big fan of this, as it can leads to some weird behavior. Are you trying to fetch the latest data from the server when you navigate to this page?
If yes it's cleaner to change the fetch policy of the query to cache and network.

Reply:
@pradel yes, the intended behaviour is to get the latest data. Thanks for the suggestion, on it.